core/src/plugins/filed/python/module/bareosfd.h

```diff
@@ -784,6 +790,16 @@ MOD_INIT(bareosfd)
   if (!pDictbIOPS) { return MOD_ERROR_VAL; }
   if (PyModule_AddObject(m, bIOPS, pDictbIOPS)) { return MOD_ERROR_VAL; }
 
+  const char* bIOPstatus = "bIOPstatus";
+  PyObject* pDictbIOPstatus = NULL;
+  pDictbIOPstatus = PyDict_New();
+  ConstSet_StrLong(pDictbIOPstatus, iostat_error, -1);
+  ConstSet_StrLong(pDictbIOPstatus, iostat_do_in_plugin, 0);
+  ConstSet_StrLong(pDictbIOPstatus, iostat_do_in_core, 1);
```

Comment:
you could re-use the C++ constants here instead of -1, 0, 1

Reply:
Makes sense!